docs/pages/pmd/userdocs/tools/ant.md

```diff
@@ -179,7 +179,7 @@ automatically and the latest language version is used.
     <target name="pmd">
         <taskdef name="pmd" classname="net.sourceforge.pmd.ant.PMDTask"/>
         <pmd shortFilenames="true">
-            <ruleset>rulesets/java/design.xml</ruleset>
+            <ruleset>category/java/design.xml</ruleset>
```

Comment:
We don't recommend including whole categories though. The recommended starting point is `rulesets/java/quickstart.xml`

Reply:
I'm fine with any that is not causing deprecation warning, so will use quickstart here.
I observed that java-basic just line below also gives:
```
...
Use Rule name category/java/bestpractices.xml/CheckResultSet instead of the deprecated Rule name rulesets/java/basic.xml/CheckResultSet. PMD 7.0.0 will remove support for this deprecated Rule name usage.
Use Rule name category/java/codestyle.xml/ExtendsObject instead of the deprecated Rule name rulesets/java/basic.xml/ExtendsObject. PMD 7.0.0 will remove support for this deprecated Rule name usage.
...
```
Could you suggest something else instead?

Reply:
Maybe then, `java-basic` should be replaced with `java-quickstart` and `rulesets/java/design.xml` with a dummy `config/my-ruleset.xml`? Quickstart is the only non-deprecated ruleset we have as of now. Or maybe one of those two should be removed. The form `language-ruleset` has become not so useful since we moved rules into categories, and reduced the number of rulesets, so maybe this has to go. @adangel wdyt?

Reply:
yes, we should just remove the example "java-basic". I don't think we have documented this notation anywhere.

I'll remove it and fix the remaining examples, which still use old rulesets....